python/batch-predictor/spec/prediction_job.proto

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 The Merlin Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package merlin.batch.spec;
+
+option go_package = "github.com/caraml-dev/merlin-pyspark-app/pkg/spec";
+option java_multiple_files = true;
+option java_outer_classname = "PredictionJobProto";
+option java_package = "com.gojek.merlin.batch.spec";
+
+message PredictionJob {
```

Comment:
LOL thanks for reverse engineering this! 🙏🥲

Reply:
Not much reverse engineering, just copied it from the file that terance deleted haha